server/command.go

```diff
@@ -484,3 +485,72 @@ func (p *Plugin) subscribeCommand(parameters []string, channelID string, config
 
 	return fmt.Sprintf("Successfully subscribed to %s.%s", fullPath, hookStatusMessage)
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	gitlabCommand := model.NewAutocompleteData("gitlab", "[command]", "Avaliable commands: connect, disconnect, todo, subscribe, unsubscribe, me, settings, webhook")
```

Comment:
```suggestion
	gitlabCommand := model.NewAutocompleteData("gitlab", "[command]", "Available commands: connect, disconnect, todo, subscribe, unsubscribe, me, settings, webhook")
```

Reply:
@aaronrothschild Thanks for the spell-check.